PSKoans/Koans/Cmdlets 1/AboutPSProviders.Koans.ps1

```diff
@@ -134,7 +134,7 @@ Describe 'Environment Provider' {
 
 Describe 'FileSystem Provider' {
     BeforeAll {
-        $Path = 'TestDrive:' | Join-Path -ChildPath 'File001.tmp'
+        $Path = 'TEMP:' | Join-Path -ChildPath 'File001.tmp'
```

Comment:
`TEMP:` as a pre-existing PSDrive doesn't exist until PS6 or 7 I think?

`TestDrive:` is provided by Pester and cleans up after itself without us having to do any manual work to do so. Was this one not working for you? 🙂